storybook/src/components/LinkList/LinkList.stories.tsx

```diff
@@ -6,40 +6,63 @@
 import { LinkList } from '@amsterdam/design-system-react/src'
 import * as Icons from '@amsterdam/design-system-react-icons'
 import { Meta, StoryObj } from '@storybook/react'
+import { contrastColorDecorator, inverseColorDecorator } from '../shared/decorators'
 import { exampleLinkList } from '../shared/exampleContent'
 
-const links = exampleLinkList()
+const linkList = exampleLinkList()
 
 const meta = {
   title: 'Components/Navigation/Link List',
   component: LinkList,
+} satisfies Meta<typeof LinkList>
+
+export default meta
+
+const linkMeta = {
+  component: LinkList.Link,
+  args: {
+    children: linkList[0],
+    contrastColor: undefined,
+    href: '#',
+    icon: Icons.ChevronRightIcon,
+    inverseColor: undefined,
+    size: undefined,
+  },
+} satisfies Meta<typeof LinkList.Link>
+
+type Story = StoryObj<typeof meta>
+type LinkStory = StoryObj<typeof linkMeta>
+
+const StoryTemplate: Story = {
   args: {
-    children: links.map((text, index) => (
+    children: linkList.map((text, index) => (
       <LinkList.Link href="#" key={index}>
         {text}
       </LinkList.Link>
     )),
   },
-} satisfies Meta<typeof LinkList>
-
-export default meta
+}
 
-const linkMeta = {
-  title: 'Components/Navigation/Link List Link',
-  component: LinkList.Link,
+const LinkStoryTemplate: LinkStory = {
+  args: {
+    children: linkList[0],
+    contrastColor: undefined,
```

Comment:
I think you can set this to `false` and remove it from the `argTypes`. Storybook can infer the correct control. Same goes for `inverseColor`